packages/account-v2/src/pages/ConnectedApps/ConnectedApps.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
-import { useFetchConnectedApps } from '@deriv/api-v2';
+import React, { useCallback, useState } from 'react';
+import { useFetchConnectedApps, useRevokeConnectedApps } from '@deriv/api-v2';
 import { Loader } from '@deriv-com/ui';
+import { ConnectedAppsRevokeModal } from './ConnectedAppsRevokeModal';
 import { ConnectedAppsTable } from './ConnectedAppsTable';
 
 export const ConnectedApps = () => {
     const { data: connectedApps, isLoading } = useFetchConnectedApps();
+    const { mutate: revokeMutate } = useRevokeConnectedApps();
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const [selectedAppId, setSelectedAppId] = useState<number | null>(null);
+
+    const handleToggleModal = useCallback((appId: number | null = null) => {
```

Comment:
We are setting default value as null for appID. In what scenario will `handleToggleModal` be called with null value?